Mathlib/Computability/Language.lean

```diff
@@ -297,6 +299,71 @@ instance : KleeneAlgebra (Language α) :=
       rw [pow_succ, ←mul_assoc m l (l^n)]
       exact le_trans (le_mul_congr h le_rfl) ih }
 
+/-- Language `l.reverse` is defined as the set of words from `l` backwards. -/
+def reverse (l : Language α) : Language α := { w : List α | w.reverse ∈ l }
+
+lemma mem_reverse : a ∈ l.reverse ↔ a.reverse ∈ l := Iff.rfl
+
+@[simp]
+lemma reverse_mem_reverse : a.reverse ∈ l.reverse ↔ a ∈ l := by
```

Comment:
This makes more sense as
```suggestion
@[simp]
lemma mem_reverse : a ∈ l.reverse ↔ a.reverse ∈ l := Iff.rfl

lemma reverse_mem_reverse : a.reverse ∈ l.reverse ↔ a ∈ l := by
```
See eg [`Set.mem_inv`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Pointwise/Basic.html#Set.mem_inv)